test/controllers/todos_controller_test.rb

```diff
@@ -989,6 +989,17 @@ def test_url_with_slash_in_query_string_are_parsed_correctly
     assert_select "a[href='#{url}']"
   end
 
+  def test_url_with_double_hyphen_is_parsed_correctly
```

Comment:
I answered generally because I wasn't totally clear what your proposed action is. If you mean open a new PR to move the existing tests, sure, that's good refactoring work. If you mean move the test in this PR, that's a moot point because this PR is closed.